wordpress-mobile/WordPress-Android
# Issue/10640 material stats

Fixes #10640

This PR moves Stats to Material Theme and adds Dark Theme support (Widgets will follow).

Stats are pretty recent, so there are very little code changes, mostly visual stuff.

Stats consists of a bunch of similar-looking components, so I tried to present most of them in the screenshots. Tons of small changes here, so I recommend doing a design review with installed APK.

![comparison_stats](https://user-images.githubusercontent.com/728822/72576593-fece6880-3884-11ea-9b27-d0e89632a849.png)


To test:
- Using a site with some amount of traffic, navigate to Stats screen and confirm the visual and functional integrity of each element in both Light and Dark themes. I suggest enabling all the Insights from the insights management screen.
- On a self-hosted site without Jetpack, navigate to Stats screen, and confirm that Jetpack install flow looks as expected.
The theme can be changed from App Settings.

PR submission checklist:
